### PR TITLE
chore: clang-tidy modernize-use-using

### PR DESCRIPTION
**Pull request by paulgessinger**
Reduce clang-tidy warnings

### CODE REVIEW

File: Fatras/src/Utilities/LandauDistribution.cpp [hunk 1]
```diff
@@ -179,10 +179,12 @@ double ActsFatras::LandauDistribution::quantile(double z) {
...
       51.005773, 53.437996, 56.123356, 59.103894};
 
-  if (z <= 0)
+  if (z <= 0) {
     return -std::numeric_limits<double>::infinity();
-  if (z >= 1)
+  }
+  if (z >= 1) {
     return std::numeric_limits<double>::infinity();
+  }
```
Comment (AJPfleger) [inline Fatras/src/Utilities/LandauDistribution.cpp:188]: This shouldn't be in this PR. Probably a duplicate from PR [#1343](https://github.com/acts-project/acts/pull/1343). But if the other one gets merged first this should vanish, I think.

**Review by AJPfleger — APPROVED** [review]
lgtm

File: Fatras/src/Utilities/LandauDistribution.cpp (hunk 1 above)
Comment (paulgessinger) [reply Fatras/src/Utilities/LandauDistribution.cpp]: Right, I think it's gone now.

**Comment by paulgessinger** [comment]
This would need reapproval, @AJPfleger.

**Review by AJPfleger — APPROVED** [review]

**Pull request merged**